feat: show enabled provider only

EurFelux opened the pull request:
<!-- Template from https://github.com/kubevirt/kubevirt/blob/main/.github/PULL_REQUEST_TEMPLATE.md?-->
<!--  Thanks for sending a pull request!  Here are some tips for you:
1. Consider creating this PR as draft: https://github.com/CherryHQ/cherry-studio/blob/main/CONTRIBUTING.md
-->

### What this PR does

<img width="1006" height="422" alt="image" src="https://github.com/user-attachments/assets/7dead73c-dfe3-4f18-8667-deada200bfeb" />

<!-- (optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: -->

Fixes #10885

### Why we need it and why it was done in this way

The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: <!-- optional: slack, other GH issue, mailinglist, ... -->

### Breaking changes

<!-- optional -->

If this PR introduces breaking changes, please describe the changes and the impact on users.

### Special notes for your reviewer

<!-- optional -->

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Documentation: A [user-guide update](https://docs.cherry-ai.com) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature.

### Release note

<!--  Write your release note:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. If no release note is required, just write "NONE".
-->

```release-note

```

EurFelux committed:
feat(provider-settings): add option to hide disabled providers

- Add hide disabled providers toggle in provider settings
- Update preference schema to store the setting
- Refactor provider list layout to accommodate new toggle

actions-user committed:
fix(i18n): Auto update translations for PR #10933

0xfullex commented on packages/shared/data/preference/preferenceSchemas.ts:
```diff
@@ -47,6 +47,7 @@ export interface PreferenceSchemas {
     'app.proxy.mode': PreferenceTypes.ProxyMode
     // redux/settings/proxyUrl
     'app.proxy.url': string
+    'app.settings.provider.hide_disabled': boolean
```
> [!NOTE]
> This comment was translated by Claude.

Preference already contains setting. This naming is semantically redundant.
Additionally, the app prefix refers to globally effective and application-wide related configurations, so it shouldn't be used here.
The correct naming should be ui.provider.show_disabled.
1. This is a UI category option related to providers.
2. We generally conventionally use affirmative rather than negative expressions for options (so use show instead of hide, and the default value needs to be adjusted accordingly).

---
<details>
<summary>Original Content</summary>

preference就是包含setting。这个命名语义重复了
此外，app前缀是指全局生效和应用整体相关的配置，这里不应该用app
正确应该是 ui.provider.show_disabled
1. 这是一个ui类别的，和provider相关的选项
2. 我们一般约定用正义而不是反义来表达选项（所以用show而不是hide，对应默认值需要调整）

</details>

EurFelux committed:
refactor(provider): change hide disabled providers to show disabled providers

Update the provider settings to use a more intuitive "show disabled" toggle instead of "hide disabled". This includes updating the preference schema, default value, and UI components to reflect this change.

EurFelux committed:
Merge branch 'feat/show-enabled-provider-only' of github.com:CherryHQ/cherry-studio into feat/show-enabled-provider-only
